src/lib/bristlefin/bristlefin.cpp

```diff
@@ -75,6 +75,14 @@ void Bristlefin::disableVout() {
   IOWrite(&BF_PL_BUCK_EN, 1); // 0 enables, 1 disables. Vout
 }
 
+void Bristlefin::sdi12Tx(){
+  IOWrite(&BF_SDI12_OE, 0); // reset to enable TX of data
+}
+
+void Bristlefin::sdi12Rx(){
+  IOWrite(&BF_SDI12_OE, 1); // set to disable TX and enable RX of data?
```

Comment:
@uma-kat please resolve the "?" on this one - does or doesn't it enable Rx?

Reply:
I shall remove the `?` since it does enable Rx